exercises/diffie-hellman/README.md

```diff
@@ -0,0 +1,59 @@
+# Diffie Hellman
+
+Diffie-Hellman key exchange.
+
+Alice and Bob use Diffie-Hellman key exchange to share secrets.  They
+start with prime numbers, pick private keys, generate and share public
+keys, and then generate a shared secret key.
+
+## Step 0
+
+The test program supplies prime numbers p and g.
+
+## Step 1
+
+Alice picks a private key, a, greater than 1 and less than p.  Bob does
+the same to pick a private key b.
+
+## Step 2
+
+Alice calculates a public key A.
+
+    A = g**a mod p
```

Comment:
Has this README been generated using configlet? There are some discrepancies I think, for example this section should be a code block.

There a description in the section on adding a new exercise in our [contributing guide](https://github.com/exercism/java/blob/master/CONTRIBUTING.md#adding-a-new-exercise) which explains how to generate the readme using configlet 🙂

Reply:
Ouch! Now i generated it using configlet 🙂
I noticed that configlet does't embed this section in a code block 🤔

Reply:
I see, I thought it was from the look of it in problem-specifications... But that's what configlet generates then it's probably right 🙂 Thanks for going to the trouble of doing that! 🙂